OvmfPkg/Include/IndustryStandard/QemuFwCfg.h

```diff
@@ -38,7 +38,7 @@
 
 //
 // The fw_cfg registers can be found at these IO Ports, on the IO-mapped
-// platforms (Ia32 and X64).
+// platform (X64).

```

Comment:
These registers are in the chipset and are not related to IA32 or X64.  I think the platforms part of this comment can be removed completely.